core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionUnitSuite.scala

```diff
@@ -77,13 +187,30 @@ class BlockManagerDecommissionUnitSuite extends SparkFunSuite with Matchers {
     try {
       bmDecomManager.start()
 
-      eventually(timeout(5.second), interval(10.milliseconds)) {
+      var previousRDDTime = Long.MaxValue
+      var previousShuffleTime = Long.MaxValue
+
+      // We don't check that all blocks are migrated because out mock is always returning an RDD.
+      eventually(timeout(10.second), interval(10.milliseconds)) {
```

Comment:
What about this timeout -- should this also be bumped up higher to 100 seconds like the one above ?

Reply:
Yeah I can do that.